tests/reporter/kubernetes.go

```diff
@@ -630,203 +626,200 @@ func (r *KubernetesReporter) logJournal(virtCli kubecli.KubevirtClient, logsdir
 	pvs, err := virtCli.CoreV1().PersistentVolumes().List(context.Background(), metav1.ListOptions{})
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to fetch pvs: %v\n", err)
+		printError("failed to fetch pvs: %v", err)
 		return
 	}
 
-	r.logObjects(virtCli, pvs, "pvs")
+	r.logObjects(pvs, "pvs")
 }
 
 func (r *KubernetesReporter) logStorageClasses(virtCli kubecli.KubevirtClient) {
 	storageClasses, err := virtCli.StorageV1().StorageClasses().List(context.Background(), metav1.ListOptions{})
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to fetch storageclasses: %v\n", err)
+		printError("failed to fetch storageclasses: %v", err)
 		return
 	}
 
-	r.logObjects(virtCli, storageClasses, "storageclasses")
+	r.logObjects(storageClasses, "storageclasses")
 }
 
 func (r *KubernetesReporter) logCSIDrivers(virtCli kubecli.KubevirtClient) {
 	csiDrivers, err := virtCli.StorageV1().CSIDrivers().List(context.Background(), metav1.ListOptions{})
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to fetch csidrivers: %v\n", err)
+		printError("failed to fetch csidrivers: %v", err)
 		return
 	}
 
-	r.logObjects(virtCli, csiDrivers, "csidrivers")
+	r.logObjects(csiDrivers, "csidrivers")
 }
 
 func (r *KubernetesReporter) logPVCs(virtCli kubecli.KubevirtClient) {
 	pvcs, err := virtCli.CoreV1().PersistentVolumeClaims(v1.NamespaceAll).List(context.Background(), metav1.ListOptions{})
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to fetch pvcs: %v\n", err)
+		printError("failed to fetch pvcs: %v", err)
 		return
 	}
 
-	r.logObjects(virtCli, pvcs, "pvcs")
+	r.logObjects(pvcs, "pvcs")
 }
 
 func (r *KubernetesReporter) logDeployments(virtCli kubecli.KubevirtClient) {
 	deployments, err := virtCli.AppsV1().Deployments(flags.KubeVirtInstallNamespace).List(context.Background(), metav1.ListOptions{})
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to fetch deployments: %v\n", err)
+		printError("failed to fetch deployments: %v", err)
 		return
 	}
 
-	r.logObjects(virtCli, deployments, "deployments")
+	r.logObjects(deployments, "deployments")
 }
 
 func (r *KubernetesReporter) logDaemonsets(virtCli kubecli.KubevirtClient) {
 	daemonsets, err := virtCli.AppsV1().DaemonSets(flags.KubeVirtInstallNamespace).List(context.Background(), metav1.ListOptions{})
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to fetch daemonsets: %v\n", err)
+		printError("failed to fetch daemonsets: %v", err)
 		return
 	}
 
-	r.logObjects(virtCli, daemonsets, "daemonsets")
+	r.logObjects(daemonsets, "daemonsets")
 }
 
 func (r *KubernetesReporter) logVolumeSnapshots(virtCli kubecli.KubevirtClient) {
 	volumeSnapshots, err := virtCli.KubernetesSnapshotClient().SnapshotV1().VolumeSnapshots(v1.NamespaceAll).List(context.Background(), metav1.ListOptions{})
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to fetch volume snapshots: %v\n", err)
+	if errors.IsNotFound(err) {
+		// The cluster doesn't support volume snapshots
```

Comment:
I am not sure, but maybe we can `printInfo` this?

Reply:
Yeah, I guess we could. I originally started this PR because I was running `make functest` repetitively and got tired of seeing the same "failures" over and over.
I wouldn't mind something phrased as positive/neutral instead of negative, like "Skipping volume snapshot collection", but is there really a good reason to show such a message?

Reply:
Yes, but the reporter collects artifacts only if there are any failures. Anyway, not a blocking issue, since I am the first one who is not convinced about adding this.

Reply:
I'll add the print from the comment above if you like it! I don't mind a message, I just don't want it to yell at me :)

Reply:
No strong opinion :) As you prefer. You can also leave things as is

Reply:
Done, PTAL!